scanner/config.go

```diff
@@ -236,6 +237,10 @@ func runScanner(detector scanners.ScannerInterface, searchDir string) scannerOut
 
 	options, projectWarnings, icons, err := detector.Options()
 	output.AddWarnings(optionsFailedTag, []string(projectWarnings)...)
+	for _, warning := range projectWarnings {
+		data := detectorErrorData(detector.Name(), errors.New(warning))
+		analytics.LogWarn(optionsFailedTag, data, "%s detector Options warning", detector.Name())
```

Comment:
is this connected to this task?
it seems in the ios/utils.go you will not add icon related error to the returned warning, so as far as I see, these warnings will not contain icon related errors.

Reply:
Yes it is not connected. Can remove it, or move it to other PR.